### PR TITLE
Remove default secrets

### PR DESCRIPTION
**Pull request by nathan-clark**
yarn tests fails w/o postcode-info-address-lookup-token if APPINSIGHTS_INSTRUMENTATIONKEY is not mapped the app wont start.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: config/default.yaml [hunk 1]
```diff
@@ -29,7 +29,8 @@ appInsights:
   roleName: "ccd-api-gateway"
 secrets:
   ccd:
-    ccd-api-gateway-oauth2-client-secret: ccd_gateway_secret
+    ccd-api-gateway-oauth2-client-secret: ""
```
Comment (smathangi) [inline config/default.yaml:32]: i think app start-up is still not failing (yarn start) without secret values

File: config/default.yaml [hunk 2]
```diff
@@ -33,7 +33,8 @@ appInsights:
...
     postcode-info-address-lookup-token: AA
-    microservicekey-ccd-gw: AAAAAAAAAAAAAAAC
-    AppInsightsInstrumentationKey: "some-key"
\ No newline at end of file
+    microservicekey-ccd-gw: ""
```
Comment (timja) [inline config/default.yaml:38]: Does the app still start up with this? the intent is that it fails to start not that it starts with an empty value.

You can work around test issues by setting NODE_ENV to be test and then loading dummy values there

Comment (mario-paniccia) [reply config/default.yaml:38]: @nathan-clark can you have a look at @timja comment please

File: yarn-audit-known-issues [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
-https://npmjs.com/advisories/1084
-https://npmjs.com/advisories/1325
-https://npmjs.com/advisories/1179
+https://npmjs.com/advisories/1316
+https://npmjs.com/advisories/1324
+https://npmjs.com/advisories/1084
+https://npmjs.com/advisories/1325
+https://npmjs.com/advisories/1179
+
+https://npmjs.com/advisories/1523
```
Comment (smathangi) [inline yarn-audit-known-issues:7]: 1523 (low severity) - No fix is currently available

Comment (timja) [reply yarn-audit-known-issues:6]: bit weird that the diff is showing up all the other lines =/

**Comment by smathangi** [comment]
> Does ccd-docker need any changes?

It has required environment variables with default values already.

**Pull request merged**